.github/workflows/e2e_android.yml

```diff
@@ -98,8 +98,8 @@ jobs:
       - name: 'Get Matrix Value'
         id: matrix_value
         run: |
-          if [[ "${{ inputs.run_full_test }}" == "true" ]] || [[ ${{ github.event_name }} == 'schedule' ]] || [[ "${{ inputs.tests_to_run}}" != "" ]] || [[ "${{ github.event.pull_request.user.login }}" == "dependabot[bot]" ]]; then
-            if [[ "${{ inputs.run_full_test }}" == "true" ]] || [[ ${{ github.event_name }} == 'schedule' ]] || [[ "${{ github.event.pull_request.user.login }}" == "dependabot[bot]" ]]; then
+          if [[ "${{ inputs.run_full_test }}" == "true" ]] || [[ ${{ github.event_name }} == 'schedule' ]] || [[ "${{ inputs.tests_to_run}}" != "" ]] || [[ "${{ github.event.push.actor.login }}" == "dependabot[bot]" ]]; then
```

Comment:
This will only work when Dependabot initially opens the PR, but won't work for subsequent commits to the branch made by other people. 

Instead of updating this, I think the paths in the `pull_request` trigger need to be updated to include `VAMobile/package.json`. Then the `[[ "${{ github.event.pull_request.user.login }}" == "dependabot[bot]" ]]` logic should be true whenever a commit by anyone is pushed to a Dependabot branch